apps/site/src/app/(main)/(home)/sections/FAQ/FAQAccordionDesktop.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+
+import SpeechDesktop from "./assets/speech-desktop.svg";
+
+import ListItemButton from "./components/ListItemButton";
+import TriangleIcon from "./components/TriangleIcon";
+
+interface FAQAccordion {
+	faq: {
+		_key: string;
+		question: JSX.Element;
+		answer: JSX.Element;
+	}[];
+}
```

Comment:
Suggestion: because this interface `FAQAccordion` is also used in `FAQAccordionMobile`, there will be less duplicate code if you moved this interface to `FAQ` and imported the interfaces as needed